src/vscode-bicep/schemas/bicepconfig.schema.json

```diff
@@ -625,6 +598,37 @@
         },
         "userDefinedFunctions": {
           "type": "boolean"
+        },
+        "prettyPrinting": {
+          "type": "boolean"
+        }
+      }
+    },
+    "formatting": {
+      "type": "object",
+      "description": "The Bicep format settings",
+      "properties": {
+        "indentKind": {
+          "type": "string",
+          "description": "The indentation kind",
+          "enum": ["Space", "Tab"]
+        },
+        "newlineKind": {
+          "type": "string",
+          "description": "The newline kind",
+          "enum": ["LF", "CRLF", "CR"]
```

Comment:
>CR [](http://example.com/codeflow?start=34&length=2)

I forget if I asked this earlier, but does our parser allow CR for a newline?

Reply:
It does. Our Lexer checks for `\n` and `\r` when scanning newline characters.

Reply:
Sounds good.